test/test_dependencies.py

```diff
@@ -58,7 +60,11 @@ def test_dummy_ros2_pkg(tmpdir):
     platform = Platform(arch='aarch64', os_name='ubuntu', ros_distro='dashing')
     out_script = ws / rosdep_install_script(platform)
 
-    gather_rosdeps(client, platform, workspace=ws)
+    customizations = PipelineStageConfigOptions(False, [], None, None, None)
```

Comment:
nit: would be good to explain what these are. ex.
```py
skip_rosdep_collection = False
skip_rosdep_keys = []
...
customizations = PipelineStageConfigOptions(
  skip_rosdep_collection,
  skip_rosdep_keys,
  ...
)
```

Or just a comment that it's a default config or something.